src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
I've seen it being used directly on a static_cast expression (e.g. here https://github.com/qt/qtbase/blob/6bceb4a8a9292ce9f062a38d6fe143460b54370e/tests/auto/corelib/tools/qscopedpointer/tst_qscopedpointer.cpp#L200) and what you find extremely often is this:

```
int foo = bar();
Q_UNUSED(foo);
```
IMHO you can shorten that by directly using it on the rvalue of the function return, but in either case, that's not a function parameter.